web/src/App.js

```diff
@@ -5,15 +5,20 @@ import './App.css';
 class App extends Component {
   constructor(props) {
     super(props)
-    this.state = {Resources: []}
+    this.state = {
+      Message: '',
+      View: {Resources: []},
+    }
   }
 
   render() {
     let el = null
-    if (!this.state.Resources || !this.state.Resources.length) {
-      el = <LoadingScreen />
+    let view = this.state.View
```

Comment:
i struggled a bit with whether to make the capitalization rules consistent with Go or with JS and decided to try to use the Go conventions for now...tho could easily be convinced otherwise...

Reply:
I'm fine with this decision! I'll let you know if I develop a stronger opinion later on.